tests/api/test_evaluators.py

```diff
@@ -15,7 +15,7 @@
 #  License along with this library.
 import pytest
 from mock import patch
-import octobot_evaluators.api.evaluators
+import octobot_evaluators.evaluators.evaluator_factory as evaluator_factory
```

Comment:
This test was about the API, if it doesn't test the API anymore I think we should move it somewhere else

Reply:
right!